.env.example

```diff
@@ -0,0 +1,2 @@
+token=yourgithubtoken
```

Comment:
So, the `.env` approach seems interesting, and I started by generating a `.env` file - but this file seems to not be used at all... perhaps you are missing some dependency?

Reply:
I tried using it in another workstation using Windows and a third one using Ubuntu 20.04. It worked after I made readme more clear. Could you try again?

Reply:
This token should be optional, in my opinion.
I tried running the code without it and it kept exiting with:
```
Environment variables are not set, please use command gitHubUsers setenv -t yourtoken -d yourdatabaseurl
```

Reply:
Actually I'm using this token cause gitHubApi has a rate limit.